### PR TITLE
Add user input delay info

### PR DESCRIPTION
**Pull request by ehanley324**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/layout-instability/pull/12.html" title="Last updated on Jun 25, 2019, 5:50 PM UTC (4ff8dd8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/layout-instability/12/ccf1cb9...4ff8dd8.html" title="Last updated on Jun 25, 2019, 5:50 PM UTC (4ff8dd8)">Diff</a>

### CODE REVIEW

**Comment by yoavweiss** [comment]
@ehanley324 - can you join the WICG and link your GH account to your W3C one to appease the IPR bots?

File: index.bs [hunk 1]
```diff
@@ -222,4 +228,3 @@ Security & privacy considerations {#priv-sec}
 
 Layout instability bears an indirect relationship to <a href="https://w3c.github.io/resource-timing/">resource timing</a>, as slow resources could cause intermediate layouts that would not otherwise be performed. Resource timing information can be used by malicious websites for <a>statistical fingerprinting</a>.
 The layout instability API only reports instability in the current browsing context. It does not directly provide any aggregation of instability scores across multiple browsing contexts. Developers can implement such aggregation manually, but browsing contexts with different <a href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin">origins</a> would need to cooperate to share instability scores.
-
```
Comment (npm1) [inline index.bs]: nit: no need to remove last blankline?

File: index.bs [hunk 2]
```diff
@@ -210,6 +215,7 @@ Identify an unstable element {#sec-identify-unstable-element}
     When asked to <dfn export>identify an unstable element</dfn>, given <var>element</var> as input, run the following steps:
 
     1. If <var>element</var>’s <a>previous frame starting point</a> is null, return false.
+    1. If a <a>mousedown</a>, <a>keydown</a>, or <a>pointerup</a> user input event has happened within 500ms of the <a>current high resolution time</a>, return false.
```
Comment (npm1) [inline index.bs]: Can we make this more clear? I would say this requires keeping track of 'last important event time' and then here we can compare with the current time. But even if we omit this and assume that the information is known, it's not clear which timestamp is being used for when those events 'happened'. Is it the event.timeStamp?

**Comment by npm1** [comment]
Also high-level comment: @skobes wasn't the plan to expose a boolean bit / timestamp of the user input instead of altogether not exposing entries that are close to user input?

**Comment by skobes** [comment]
> Also high-level comment: @skobes wasn't the plan to expose a boolean bit / timestamp of the user input instead of altogether not exposing entries that are close to user input?

Yes that is the plan.  Maybe we should spec that instead of the current behavior?

**Comment by ehanley324** [comment]
Sounds like we want to spec somethign slightly different with the boolean bit and timestamp.  Going to close this pull request since it won't land as is.

**Pull request closed without merging**